clean_up.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+set -e
+
+export IGNORE_NETPOL=("acp-deny-all")
+export IGNORE_CONFIGMAP=("bundle")
+export IGNORE_SECRET=("file-vault-cert" "firearms-notprod-s3" "icasework" "keycloak" "keycloak-form" "postcode-auth" "redis" "s3-bucket" "ses" "session-secret")
```

Comment:
Update names

Reply:
this has been refactored now and isn't included